tags/line_commands/line_commands.talon

```diff
@@ -74,3 +74,6 @@ select camel left: user.extend_camel_left()
 select camel right: user.extend_camel_right()
 go camel left: user.camel_left()
 go camel right: user.camel_right()
+
+go char <user.unmodified_key>: user.jump_cursor_to_next_char(unmodified_key)
```

Comment:
I'm not sure these should live in line commands. Maybe they make sense alongside the plugin and be enabled with the appropriate tag? As it is, this would add some commands that won't work in a lot of applications

Reply:
hrm. I do think there is the potential for this to be used elsewhere in community, though not sure of any good example where to add it off the top of my head. But for instance, I already used it in my repo for zsh line editor (would like eventually to PR that), and was going to adapt vim to use it after (though that won't become part of community) and similarly I guess people could add it for emacs (maybe there is no emacs in community). I added a separate vscode example in this PR to community specifically to justify it since we didn't want an otherwise completely dead command, but I do see your point.. 

Having it on the plugin side only doesn't entirely make sense due to the potential to be used inside community, but if you guys decide you don't want it in here and just want to lift the plugin stuff from this PR instead I'm good with that. Can always add it later once there is a second use case.